cloudformationTemplates/urlPreviewForAsyncChat/cloudformation.yaml

```diff
@@ -453,6 +488,7 @@ Resources:
 
   s3BucketPolicy:
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

S3_BUCKET_SSL_REQUEST_ONLY : S3 buckets do not have built-in SSL/TLS support. Make sure that effect is set to deny and SecureTransport is set to false. This ensures that all communication between the client and CloudFront, as well as between CloudFront and your S3 bucket, is encrypted.